net-misc/eventd/eventd-0.21.0.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit linux-info systemd
+
+DESCRIPTION="A small daemon to act on remote or local events"
+HOMEPAGE="https://www.eventd.org/"
+SRC_URI="https://www.eventd.org/download/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ LGPL-3+ MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug fbcon +introspection ipv6 libcanberra libnotify +notification
+	purple sound speech ssdp systemd test websocket +X zeroconf"
+
+REQUIRED_USE="
+	X? ( notification )
+	fbcon? ( notification )
+	notification? ( || ( X fbcon ) )
+"
+
+COMMON_DEPEND="
+	dev-libs/glib:2
+	sys-apps/util-linux
+	introspection? ( dev-libs/gobject-introspection )
+	libcanberra? ( media-libs/libcanberra )
```

Comment:
Depending on the package this should probably not be optional, libcanberra is a very small package (this should be in https://wiki.gentoo.org/wiki/Project:GNOME/Gnome_Team_Ebuild_Policies but for some reason it isn't yet). Here is it probably ok to have it but your might consider it odd when there is a sound USE flag next to it that only does pulseaudio (on first sight). Maybe they should go together ?